src/Agent/NewRelic/Agent/Core/Aggregators/TransactionTraceAggregator.cs

```diff
@@ -33,6 +33,19 @@ public TransactionTraceAggregator(IDataTransportService dataTransportService, IS
 
         protected override bool IsEnabled => _configuration.TransactionTracerEnabled;
 
+        protected override TimeSpan HarvestCycle
+        {
+            get
+            {
+                if (_configuration.TransactionTracesHarvestCycle.HasValue)
+                {
+                    return _configuration.TransactionTracesHarvestCycle.Value;
+                }
+
+                return DefaultHarvestCycle;
```

Comment:
Should we move the default value into DefaultConfiguration so that the logic for the harvest cycle is in just one location?